admin/client/src/components/MessageBox/MessageBox.js

```diff
@@ -0,0 +1,105 @@
+import React, { PropTypes } from 'react';
+import SilverStripeComponent from 'lib/SilverStripeComponent';
+import { Alert } from 'react-bootstrap-ss';
+import castStringToElement from 'lib/castStringToElement';
+
+/**
+ * A wrapper for Alert messages in react-bootstrap.
+ * Displays a given message.
+ */
+class MessageBox extends SilverStripeComponent {
```

Comment:
Why are we calling this a `MessageBox` when it's just wrapping an `Alert` bootstrap component? I think if we're just creating "better alert", the file path is enough to convey that. `import Alert from 'components/Alert/Alert'` vs. `import { Alert } from 'react-bootstrap-ss'`. We've used the same approach for `components/PopoverField/PopoverField` vs. `import { Popover } from 'react-bootstrap-ss'`